### PR TITLE
Update jenkins after sdk merge - Closes #3350

### LINKED ISSUE #3350: Jenkins for sdk need to be fixed according to the new structure
### Expected behavior
Tests from lisk-framework(PR and network), lisk-elements, lisk-commander runs separately but all for each PRs

### Actual behavior
Jenkins tests are not working 

### Steps to reproduce
Run Jenkins

### Which version(s) does this affect? (Environment, OS, etc...)
development-

### PR DESCRIPTION
**Pull request by ghost**
### What was the problem?

Jenkins configuration needed to be overhauled after the SDK merge.

### How did I fix it?

Refactored `Jenkinsfile` according to requirements by @shuse2 and @ManuGowda 

### How to test it?

Look at 
- https://jenkins.lisk.io/job/lisk-sdk/ and
- https://jenkins.lisk.io/job/lisk-sdk-framework/
and ensure the right tests are running (some tests might need fixing outside of this PR)

### Review checklist

* The PR resolves #3350 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: Jenkinsfile.sdk [hunk 1]
```diff
@@ -0,0 +1,47 @@
+@Library('lisk-jenkins') _
+
+pipeline {
+	agent { node { label 'lisk-commander' } }
```
Comment (ManuGowda) [inline Jenkinsfile.sdk]: `lisk-sdk`?

**Review by shuse2 — DISMISSED** [review]

File: framework/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+		string(name: 'LOG_LEVEL', defaultValue: 'error', description: 'To get desired build log output change the log level', ),
+		string(name: 'FILE_LOG_LEVEL', defaultValue: 'error', description: 'To get desired file log output change the log level', ),
+		// used by tests
+		string(name: 'LOG_DB_EVENTS', defaultValue: 'false', description: 'To get detailed info on db events log.', ),
+		string(name: 'SILENT', defaultValue: 'true', description: 'To turn off test debug logs.', )
+	 ])
+])
+
+pipeline {
+	agent { node { label 'lisk-core' } }
```
Comment (ManuGowda) [inline framework/Jenkinsfile]: `lisk-framework`?

**Review by ManuGowda — COMMENTED** [review]
LGTM 👍

File: Jenkinsfile.sdk (hunk 1 above)
Comment (fchavant) [reply Jenkinsfile.sdk]: This is just a label be we could change it.

**Review by ManuGowda — APPROVED** [review]

**Review by shuse2 — APPROVED** [review]

**Pull request merged**